Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -169,11 +169,12 @@ function Main {
 			} else {
 				Write-LogInfo "VM is still resuming!"
 			}
-		}
-		if ($vmCount -le 0){
-			Write-LogInfo "VM resume completed"
-		} else {
-			Throw "VM resume did not finish"
+			if ($vmCount -le 0){
+				Write-LogInfo "VM resume completed"
+				break
+			} else {
+				throw "VM resume did not finish"
```

Comment:
I didn't understand why do we need a 'while loop' as the execution of if-else won't continue this loop?

Reply:
while-loop is for timer. if-statement is for exiting in advacne.

Reply:
Fixed in 72ab0e9994c123bd3c098ee401944b4eae5fe02f